packages/core/src/components/Menu/MenuItem/MenuItem.tsx

```diff
@@ -90,7 +90,7 @@ export interface MenuItemTitleComponentProps extends Omit<MenuItemProps, "title"
 
 const MenuItem: VibeComponent<MenuItemProps | MenuItemTitleComponentProps> & {
   iconType?: typeof Icon.type;
-  tooltipPositions?: typeof DialogPosition;
+  tooltipPositions?: TooltipPositionsType;
```

Comment:
this can be removed

Reply:
you removed the static prop from here
```
export default withStaticProps(MenuItem, {
  iconType: Icon.type,
  tooltipPositions: DialogPosition
});
```

so either get it back there, or remove line 93